person.rb

```diff
@@ -1,5 +1,10 @@
-class Person
+require './nameable'
+require './trimmer_decorator'
+require './capitalize_decorator'
+
```

Comment:
There are a couple of issues with your `person.rb` file:
1. It is best to use `require_relative` instead of `require` when loading files within the same directory. 

In Ruby, both `require` and `require_relative` are used to load external files or libraries into your Ruby program. However, they have a slight difference in terms of how they locate and load the files.

`require`: `require` is used to load files that are part of Ruby’s load path (`root path`) or are installed as gems. It searches for the file in the directories specified in the `$LOAD_PATH` variable. If it finds the file, it loads it into memory and returns `true`. If it doesn’t find the file, it raises a `LoadError`.

![image](https://github.com/Shaheryar0054/School-library/assets/38088818/1bbf3700-f626-402d-bb40-1a8c74f3c251)

`require_relative`: `require_relative` is used to load files that are `relative to the current file’s directory`. It searches for the file in the `same directory` as the file containing the `require_relative` statement. If it finds the file, it loads it into memory and returns `true`. If it doesn’t find the file, it raises a `LoadError`.

2. You have a typo in your file naming. The `person.rb` file requires `trimmer_decorator`, yet the file name is `timmer_decorator`. Kindly rename the filename so that it is consistent with the required statement. You are missing the `r` in `trimmer`, you have `timmer` instead. This is causing the following error when testing your application.

![image](https://github.com/Shaheryar0054/School-library/assets/38088818/67ff5ea3-0776-4f28-9a89-0395e9605559)

Otherwise, with the errors fixed your application pass all the tests. Great job!!! 🚀 

![image](https://github.com/Shaheryar0054/School-library/assets/38088818/d001f164-6bf2-4d86-9af3-57e8be733fad)